tests/integration/deploy-operator.sh

```diff
@@ -0,0 +1,74 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+install-operator-k8s() { # installs istio-operator on kubernetes
+    local ROOT
+    ROOT="$(git rev-parse --show-toplevel)"
+    local TAG
+    TAG="${TAG:-$(git rev-parse HEAD)}"
+    local NS="${NS:-openshift-operators}"
+    # var name IMAGE is in kind provisioner
+    local OPERATOR_IMAGE="${OPERATOR_IMAGE:-localhost:5000/istio-operator-integ}:${TAG}"
+    local ISTIO_CNI_IMAGE_NAME="${ISTIO_CNI_IMAGE_NAME:-quay.io/maistra-dev/istio-cni-ubi8-integ:latest}"
+    local PILOT_IMAGE_NAME="${PILOT_IMAGE_NAME:-quay.io/maistra-dev/pilot-ubi8-integ:latest}"
+    local PROXY_IMAGE_NAME="${PROXY_IMAGE_NAME:-quay.io/maistra-dev/proxyv2-ubi8-integ:latest}"
+    # var name VERSION is in Makefile
+    local VERSION="2.3.0"
+
+    echo "--------------------------------"
+    echo "Installing operator"
+    echo "Using image $OPERATOR_IMAGE"
+    echo "--------------------------------"
+
+    kubectl get ns "$NS" >/dev/null 2>&1 || kubectl create namespace "$NS"
+
+    kubectl apply -f "${ROOT}/manifests-maistra/${VERSION}/servicemeshcontrolplanes.crd.yaml"
+    kubectl apply -f "${ROOT}/manifests-maistra/${VERSION}/servicemeshmemberrolls.crd.yaml"
+    kubectl apply -f "${ROOT}/manifests-maistra/${VERSION}/servicemeshmembers.crd.yaml"
+
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/rbac.yaml" | kubectl apply -n "$NS" -f -
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/serviceaccount.yaml" | kubectl apply -n "$NS" -f -
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/service.yaml" | kubectl apply -n "$NS" -f -
+
+    openssl req -x509 -newkey rsa:4096 -keyout /tmp/key.pem -out /tmp/cert.pem -sha256 -days 365 -nodes -subj "/CN=istio-operator" -addext "subjectAltName = DNS:maistra-admission-controller.$NS.svc"
+
+    kubectl create -n "$NS" secret tls maistra-operator-serving-cert --key=/tmp/key.pem --cert=/tmp/cert.pem
+    kubectl create -n "$NS" configmap maistra-operator-cabundle --from-file=service-ca.crt=/tmp/cert.pem
+
+    sed -e "s@quay.io/maistra/istio-ubi8-operator:2.3.0@${OPERATOR_IMAGE}@g" \
+        -e "s@namespace: istio-operator@namespace: $NS@g" \
+        -e "s@quay.io/maistra/istio-cni-ubi8:2.3.0@${ISTIO_CNI_IMAGE_NAME}@g" \
+        -e "s@quay.io/maistra/pilot-ubi8:2.3.0@${PILOT_IMAGE_NAME}@g" \
+        -e "s@quay.io/maistra/proxyv2-ubi8:2.3.0@${PROXY_IMAGE_NAME}@g" \
+        "${ROOT}/deploy/src/deployment-maistra.yaml" \
+        | yq-python -y 'del(.spec.relatedImages)' \
```

Comment:
You can remove this line. It's a leftover from when I used the CSV to deploy the operator.